### PR TITLE
[Merged by Bors] - feat(AlgebraicGeometry/ProjectiveSpectrum/Scheme): add the instance `IsLocalization`

### PR DESCRIPTION
**Pull request by jjaassoonn**
If `x` is a point in the basic open set `D(f)` where `f` is a homogeneous element of positive
degree, then the homogeneously localized ring `A⁰ₓ` has the universal property of the localization
of `A⁰_f` at `φ(x)` where `φ : Proj|D(f) ⟶ Spec A⁰_f` is the morphism of locally ringed space
constructed as above.

Co-authored-by: Andrew Yang <the.erd.one@gmail.com>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by erdOne** [comment]
Thanks!
maintainer merge

**Comment by joelriou** [comment]
Thanks!

bors merge

File: Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean [hunk 1]
```diff
@@ -696,6 +696,65 @@ lemma toStalk_stalkMap_toSpec (f) (x) :
...
+/--
+If `x` is a point in the basic open set `D(f)` where `f` is a homogeneous element of positive
+degree, then the homogeneously localized ring `A⁰ₓ` has the universal property of the localization
+of `A⁰_f` at `φ(x)` where `φ : Proj|D(f) ⟶ Spec A⁰_f` is the morphism of locally ringed space
+constructed as above.
+-/
+lemma isLocalization_atPrime (f) (x : pbo f) {m} (f_deg : f ∈ 𝒜 m) (hm : 0 < m) :
+    @IsLocalization (Away 𝒜 f) _ ((toSpec 𝒜 f).1.base x).asIdeal.primeCompl
+      (AtPrime 𝒜 x.1.asHomogeneousIdeal.toIdeal) _
+      (mapId 𝒜 (Submonoid.powers_le.mpr x.2)).toAlgebra := by
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean:708]: Can we use named parameters instead of `@`?

Comment (erdOne) [reply Mathlib/AlgebraicGeometry/ProjectiveSpectrum/Scheme.lean:708]: The instance argument is not named, and it is hard to use `letI` to massage lean into accepting it. We would need to provide 4 parameters out of 6 anyway.

**Pull request closed without merging**